### DISCUSSION TITLE
v0.3.0rc

### DISCUSSION THREAD

**Discussion by shesek** [shesek/spark-wallet#185] [General]
- BOLT12 offers! ✨🎉

  - Send payments using BOLT12 offers, including support for `any` amounts, quantities, fiat denomination and sending a note to the payee (`payer_note`).

  - Create reusable BOLT12 offers to receive multiple payments (basic support only).

  - Pull payments using BOLT12 withdrawal offers (`send_invoice=true`).

  - Recurrence is not supported yet -- to be added in a followup release.

  **Requires c-lightning v0.10.1+** with `experimental-offers` enabled.

- UI: Improve the main screen initial loading times

  Pay/Request can now be used immediately, without waiting for the wallet's data to load first.

- UI: Always display payment amounts in USD terms too

- UI: Improved error reporting and display

- Docker: Allow (and recommend) mounting the directory with the `lightning-rpc` socket file
  (fixes a [compatibility issue with BTCPay](https://github.com/btcpayserver/btcpayserver-docker/pull/462))

- Fix: Properly display the NoScript message on Firefox (#183)

- Update clightning-client to v0.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/shesek/spark-wallet/releases/tag/v0.3.0rc'>v0.3.0rc</a>.</em>

**Comment by btweenthebars** [comment]
I tried this prerelease version and got a lot of error complaining  "_listpayments" or "_listxxxxx" doesn't exist on UI

**Reply by shesek** [reply]
It seems like you might've updated the spark client without also updating the spark server.

Can you run `spark-wallet --version` on the server and confirm it returns v0.3.0rc?

**Reply by kalibox** [reply]
Hey @shesek, when is this rc likely to go live? Thanks and keep up the good work, my dude.

**Comment by m46nus** [comment]
Is this project dead? Any alternatives for connecting to c-lighning from android phone?
